### PR TITLE
Boilerplate & baseline. 

### PR DESCRIPTION
**Pull request by jlikesplants**

### CODE REVIEW

File: lib/market.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class FarMar::Market
+  attr_reader :market_id, :name, :address, :city, :state, :zip
+
+  def read_csv(file)
+    CSV.read(file, 'r')
```
Comment (jnf) [inline lib/market.rb:5]: This is a pretty clever solution! Reading the csv each time is pretty expensive though; I think you'll feel your tests slowdown a bit as your start rummaging through the larger files. A possible refactor would be to read the csv once, at the beginning on `initialize`, and set it to a local variable, like `raw_data = read_csv("./support/markets.csv")`.

Once you've done that, you can assign your instance variables like you're already doing, but reference the `raw_data` variable instead, like `@name = raw_data[csv_index][2]`.

File: specs/vendor_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require_relative './spec_helper'
+require_relative '../farmar'
+
+require 'simplecov'
```
Comment (jnf) [inline specs/vendor_spec.rb]: All of these simplecov lines in the specs should only happen once at the beginning of `spec_helper`. Simplecov is pretty picky about that. Please remove the simplecov lines from your individual specs and instead require and start simplecov at the very beginning of `spec_helper`.

**Comment by jnf** [comment]
Hi! I left you a couple of comments that I'd like you to look at before you move on. One is about Simplecov and must be addressed before you move onto the requirements. Please address it and let me know when I can look at it. :)

**Comment by jlikesplants** [comment]
Hi Jeremy, thanks! I fixed those and pushed it.

**Comment by jnf** [comment]
Looks great! Good job! :+1:

**Pull request closed without merging**